client/src/components/AddMovieForm/AddMovieForm.js

```diff
@@ -0,0 +1,112 @@
+import { useState } from 'react'
+import styled from 'styled-components/macro'
+import axios from 'axios'
+import ButtonGreen from '../Buttons/ButtonGreen/ButtonGreen'
+import SuccessCard from '../Messages/SuccessCard/SuccessCard'
+import ErrorCard from '../Messages/ErrorCard/ErrorCard'
+
+const AddMovieForm = () => {
+  const [tmdbId, setTmdbId] = useState('')
+  const [successMessage, setSuccessMessage] = useState('')
+  const [errorMessage, setErrorMessage] = useState('')
+
+  // Comments will be deleted later on
+  /*
+    The function has to be async, because it has to wait for the response of the tmdb api in the try catch block.
+    Once it has an response, the response can be positive or negative.
+    If the response is positive we allow (allowPostMovie) the movie to be posted to our database.
+    If the response is negative we set an error message.
+  */
+  const addMovieHandler = async event => {
+    event.preventDefault()
+    event.target.reset()
+
+    const newMovie = {
+      tmdb_id: tmdbId,
+    }
+
+    let allowPostMovie = false
+
+    try {
+      const response = await axios.get(
+        `https://api.themoviedb.org/3/movie/${tmdbId}?api_key=7a72c13d262ed45c671611991b0f6086`
```

Comment:
Uh, please _never ever_ add your API key in your frontend! People might do something weird with it and we never want to have something like that available in public. I would recommend you to:
1. Change your API key as this one is now commited and seen by other people
2. Add this `GET` request to your backend
React is _always_ running in the browser and your source code therefore visible for every other developer/user. 
To read more about it, please have a look at this article: https://unicorn-utterances.com/posts/keeping-api-keys-secret-in-react-apps/
(It is with React classes, but the content is still valid and valuable.)